docs/README.md

```diff
@@ -153,13 +153,13 @@ Sub-commands:
   * `6` or `no`: Default logs and status output turned off.
   * `7` or `onlystatus`: Default logs turned off, but status output turned on.
 * `-o` or `--output string`: Script output folder. Defaults to working folder.
-* `-r` or `--raw`: Include raw properties in structure.
+* `-r` or `--raw`: Include raw properties in the structure.
 * `--summary string`: Set the type of summary to display after the test run. Defaults to `simple`.
-  * `0` or `undefined`: Simple, prints count of objects and warnings, and all warnings.
+  * `0` or `undefined`: Simple summary, includes the number of objects and warnings and lists all warnings.
   * `1` or `none`: No summary.
-  * `2` or `simple`: Simple, prints count of objects and warnings, and all warnings.
-  * `3` or `extended`: Extended summary, includes list of all objects in structure.
-  * `4` or `full`: Currently same as `extended` summary, includes list of all objects in structure.
+  ?* `2` or `simple`: Simple summary, includes the number of objects and warnings and lists all warnings.
+  * `3` or `extended`: Extended summary, includes a list of all objects in the structure.
+  * `4` or `full`: Currently the same as the `extended` summary, includes list of all objects in structure.
```

Comment:
Maybe change to "includes a list of all objects in the structure."?
(Yes, I like "a" and "the" as well!)